### ISSUE TITLE
&feature=youtu.be stops the program

### ISSUE THREAD

**Issue by amadouderza** [deepjyoti30/ytmdl#59]
Hi,
Copy pasting URL with an ampersand after the video ID stops the program.
Removing it, it works.
(ytmdl https://www.youtube.com/watch?v=8fSpEYQUNpQ is alright)
Is it possible to trim it automatically?
Regards

```
ytmdl https://www.youtube.com/watch?v=8fSpEYQUNpQ&feature=youtu.be
[7] 26976
xxx@xxx:/ $  ==> Searching to see if already present in /home/pi/Syncthing/
 ==> Searching Youtube for https://www.youtube.com/watch?v=8fSpEYQUNpQ
 ==> Choose One
 [1] Moskau by Dschinghis Khan - Topic with dur Durée 4:31
 [2] Moskau - Dschinghis Khan (8-bit Soundfont) by NockeyNoo with dur Durée 4:19
 [3] How to easily make money on youtube by Rubix Cube with dur Durée 0:08
 [4] Moskau - Animation meme collab with SaikaTheCatto OvO by Adrieliu with dur Durée 0:55
 [5] The quadratic formula song (it's cringe) by KaijuBanjo with dur Durée 1:10
 [0] More results
 ==> Enter Choice [a valid choice] 1
bash: 1: command not found

[7]+  Stopped                 ytmdl https://www.youtube.com/watch?v=8fSpEYQUNpQ
```

**Comment by deepjyoti30** [comment]
Sure, I'll add a fix in the next release to automatically fix that.

**Comment by deepjyoti30** [comment]
Should be fixed in 3850612
